test/unit/fuzz/calculate-price/calculatePrice.tree

```diff
@@ -0,0 +1 @@
+calculatePrice.t.sol
```

Comment:
I am not sure what these tree files are supposed to be about.
```suggestion
calculatePrice.t.sol

```

Reply:
See [this](https://prberg.com/presentations/solidity-summit-2023/).

Examples: 
- [balancer cow-amm])https://github.com/balancer/cow-amm/tree/main/test/unit)
- [sablier v2](https://github.com/sablier-labs/v2-core)